sdk-core/custom_tokens.md

```diff
@@ -0,0 +1,40 @@
+## CustomToken
+
+```
+dto CustomToken:
+    // E.g. "FOO-abcdef", "EGLD".
+    identifier: string;
+    nonce: number;
+```
+
+## TokenComputer
+
+```
+class CustomTokenComputer:
+    // The constructor is not captured by the specs; it's up to the implementing library to define it.
+
+    // Returns token.nonce == 0.
+    is_fungible_token(token: Token): boolean;
+
+    // Given "FOO-abcdef-0a" returns 10.
+    extract_token_nonce_from_extended_identifier(extended_identifier: string): number;
+
+    // Given "FOO-abcdef-0a" returns "FOO-abcdef".
+    extract_token_identifier_from_extended_identifier(extended_identifier: string): string;
+
+    // Given "FOO-abcdef-0a" returns "FOO".
+    extract_token_name_from_extended_identifier(extended_identifier: string): string;
+
+    // Given "FOO-abcdef" returns "FOO".
+    extract_token_name_from_identifier(identifier: string): string;
```

Comment:
I understand the explicitly of having both methods, but maybe these can be only one method? Just a thought.

Reply:
Good point, kept one method, updated comments.